packages/preview/minimal-presentation/0.2.0/template/main.typ

```diff
@@ -0,0 +1,109 @@
+#import "@preview/minimal-presentation:0.1.0": *
```

Comment:
You should update that import, it should fix all the other warnings, that come from the older version of the package (they are just reported in the wrong place for some reason).

```suggestion
#import "@preview/minimal-presentation:0.2.0": *
```